test/unit/worker.js

```diff
@@ -70,9 +80,7 @@ describe('Worker', function () {
     beforeEach(function () {
       opts.runNow = false;
       worker = Worker.create(opts);
-      sinon.stub(error, 'log');
     });
-    afterEach(function () { error.log.restore(); });
```

Comment:
ah ha! took me a minute, but I figured out why these were here. If you run the tests w/ `LOG_LEVEL=error` (which I'm convincing myself more and more that I should just put in that `npm` command), the function below named `should catch TaskFatalError, not retry, and call done` ends up logging an error. Would you be able to stub the `Worker.log.error` and assert that it was called?